client/src/components/MapView.vue

```diff
@@ -150,6 +149,9 @@ export default defineComponent({
           trackUserLocation: true,
         }),
       );
+
+      // Add zoom in / zoom out buttons to map.
+      this.map.addControl(new mapboxgl.NavigationControl());
```

Comment:
Do you need to remove the other code you added?

Reply:
Do you mean the other zoom handler? never merged it, just closed out the PR so no need to remove.